src/content.ts

```diff
@@ -51,31 +51,30 @@ const getRegion = () => {
   return document.getElementById('awsc-mezz-region')?.getAttribute('content')
 }
 
-const configListIsYaml = (configList: string) => /^\s*[#-]/.test(configList)
-const configListIsJsonc = (configList: string) => /^\s*[\/\[]/.test(configList)
-
 const loadConfigList = async (): Promise<ConfigList | null> => {
   const configList = await configRepository.get()
   if (configList) {
-    if (configListIsYaml(configList)) {
-      return yaml.load(configList) as ConfigList
-    } else if (configListIsJsonc(configList)) {
-      return JSONC.parse(configList)
-    } else {
-      return JSON.parse(configList)
-    }
+    return parseConfigList(configList)
   } else {
     return null
   }
 }
 
+const parseConfigList = (configList: string) => {
+  try {
+    return yaml.load(configList) as ConfigList
+  } catch (e) {
+    return JSONC.parse(configList) as ConfigList
+  }
+}
+
 const loadAccountNameList = async (): Promise<AccountName[] | null> => {
   const accountNameList = await accountNameRepository.get()
   return accountNameList ? (JSON.parse(accountNameList) as AccountName[]) : null
 }
 
 const isEnvMatch = (env: Environment, accountId: string, region: string) =>
-  env.account === accountId && (env.region ? env.region === region : true)
+  String(env.account) === accountId && (env.region ? env.region === region : true)
```

Comment:
```suggestion
  env.account == accountId && (env.region ? env.region === region : true)
```

多分こっちの方が意図が明確だと思います（諸説あると思います 🤣 ）

Reply:
JSONCの書き方によっては、`env.account` が number 型で渡ってくるのですよ。
ダブルクォートを強制するのも難しいと思ったのでキャストしました🙏

Reply:
`==`だと`===`と違って型が曖昧になるのでそれを利用したコードです

Reply:
あ、すみません...サッと見て `===` に空目してました。